client/components/button/style.scss

```diff
@@ -123,34 +123,41 @@ button {
 // Primary buttons
 .button.is-primary {
 	background: var( --color-accent );
-	border-color: var( --button-primary-border-color );
+	border-color: var( --color-accent-dark );
 	color: $white;
 
 	&:hover,
 	&:focus {
-		border-color: var( --color-primary-dark );
+		border-color: var( --color-accent-dark );
 		color: $white;
 	}
+
+	.accessible-focus &:focus {
+		box-shadow: 0 0 0 2px var( --color-accent-light );
```

Comment:
[According to @drw158](https://github.com/Automattic/wp-calypso/pull/29443#issuecomment-448075590) we're going to use a slightly lighter background instead of coloring the border on hover. See e.g. [here for code comparison](https://github.com/Automattic/wp-calypso/pull/29443/files#diff-83fbe8f6fe8f918b282e2397d32c7d28R184).

Reply:
thought: maybe we should distinguish between `focus` and `hover` and use both the slightly lighter background for hover and the focus ring as you suggested in this PR 🤔  /cc @drw158